internal/scheduler/submitcheck.go

```diff
@@ -78,7 +79,18 @@ func NewSubmitChecker(
 
 func (srv *SubmitChecker) Run(ctx context.Context) error {
 	srv.updateExecutors(ctx)
-	ticker := time.NewTicker(1 * time.Minute)
+
+	var ticker *time.Ticker
```

Comment:
This is now an env var that can be controlled. It defaults to 1m, but can be changed as required.

I imagine this is a usecase that users would actually care about? So I think it's fine?

Reply:
Yea but the tests are testing this prod code with 1s while in prod we are using with a minute.  That could cause problems where our testing is not actually testing the real code.  Hence why it is usually bad practice to modify code for testing.  

What does this field do?  We could maybe have a separate PR for this but this seems to be unrelated to the testing code at the moment?

Reply:
This is literally just a wait sequence for how often to check for executor clusters. Without it, the server waits 1 minute for the executor to connect. 

This is why there needed to be a sleep before, but with this change there doesn't anymore. This was also the change that Albin and Chris suggested to make for speedup.

Reply:
Kk.  I guess I worry that this may make it even more fickle..  

I get this error pretty often with the 1 minute timelime but as long as it wasn't a driveby change I'm happy.